src/lib/Metadata/Metadata.js

```diff
@@ -1,134 +1,113 @@
-/*!
- * Copyright(c) ConsenSys Software Inc.
- * Copyright(c) https://consensys.net/
- * MIT Licensed
- */
-
-import { readFileSync } from 'fs';
-
-import IPFS from '../../services/ipfsService.js';
-import { ERROR_LOG, errorLogger } from '../error/handler.js';
+import { errorLogger, ERROR_LOG } from '../error/handler.js';
+import {
+  contractMetadataSchema,
+  freeMetadataSchema,
+  tokenMetadataSchema,
+} from './metadata.schema.js';
 
 export default class Metadata {
-  IPFS_BASE_URL = 'https://ipfs.io/ipfs/';
+  /**
+   * Create token metadata
+   * @param {string} name Name of the token
+   * @param {string} description description of the token
+   * @param {string} image tokenURI
```

Comment:
image is more a path to your local FS or a link to a remote image, isn't it?

Reply:
tokenURI or contractURI are a path to the metadata file btw

Reply:
nope it's our tokenURI

Reply:
Maybe the spec is wrong.. `image` is the link to the image.

Look at this https://docs.opensea.io/docs/metadata-standards#metadata-structure